tests/integration/integration_test.go

```diff
@@ -1480,28 +1480,15 @@ func TestPulumiInstallInstallsPackagesIntoTheCorrectDirectory(t *testing.T) {
 	e.RunCommand("pulumi", "up", "--non-interactive", "--skip-preview")
 }
 
-func TestOverrideComponentNameAndNamespace(t *testing.T) {
-	t.Parallel()
-
+func TestOverrideComponentNamespace(t *testing.T) {
 	e := ptesting.NewEnvironment(t)
 	defer e.DeleteIfNotFailed()
 
-	cwd, err := os.Getwd()
+	t.Setenv("PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION", "false")
+	stdout, _ := e.RunCommand("pulumi", "package", "get-schema",
+		"github.com/pulumi/component-test-providers/test-provider@b39e20e4e33600e33073ccb2df0ddb46388641dc")
+	var packageSpec schema.PackageSpec
+	err := json.Unmarshal([]byte(stdout), &packageSpec)
 	require.NoError(t, err)
-
-	e.ImportDirectory("component_test")
-	e.RunCommand("pulumi", "login", "--cloud-url", e.LocalURL())
-	e.CWD = filepath.Join(e.RootPath, "python-component")
-	e.RunCommand("python", "-m", "venv", "venv")
-	pythonBin := "./venv/bin/python"
-	if runtime.GOOS == "windows" {
-		pythonBin = ".\\venv\\Scripts\\python.exe"
-	}
-	e.RunCommand(pythonBin, "-m", "pip", "install", "-r", "requirements.txt")
-	e.RunCommand(pythonBin, "-m", "pip", "install", "-e", filepath.Join(cwd, "..", "..", "sdk", "python"))
-
-	e.CWD = filepath.Join(e.RootPath, "ts-consumer")
-
-	stdout, _ := e.RunCommand("pulumi", "install")
-	require.Contains(t, stdout, "import * as myComponent from \"@overridden-namespace/my-component\"")
+	require.Equal(t, "pulumi", packageSpec.Namespace)
```

Comment:
This doesn't test the override, right? We probably need to consume a component from a non-pulumi org and test its namespace.

Reply:
Added a comment that explains it.